docs/contents/proposals/secrets/secrets.md

```diff
@@ -1,22 +1,95 @@
 # Secrets Integration Design Doc
 
 ## Problem Description
-Alice is a frequent Kubernetes user excited to create a database instance using Amazon RDS. She is very familiar with Kubernetes, but doesn't want to dive into learning about the Amazon API space. She is thrilled to use ACK, AWS Controllers for Kubernetes, to easily create a MySQL database. Before ACK, Alice needed to manually call the createDBInstance method from the RDS API in the AWS console, and type in a MasterUserPassword in plain text. With the method call to create and the method call to describe the DB instance, her password was in plain text for the API request and response. That didn't seem very secure to Alice. 
-The Custom Resource Definition currently shows that the MasterUserPassword is a string:
 
-![Initial CRD for Desired State of DB Instance](images/rds-db-instance-crd-before.png)
+#### Before ACK
+Alice is a frequent Kubernetes user excited to create a database instance using [Amazon RDS](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/Welcome.html). She is very familiar with Kubernetes, but doesn't want to dive into learning about the Amazon API space. She is thrilled to use [ACK, AWS Controllers for Kubernetes](https://github.com/aws/aws-service-operator-k8s), to easily create a MySQL database. Before ACK, Alice needed to manually call the [createDBInstance](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html) method from the RDS API in the AWS console, and type in a MasterUserPassword in plain text. 
+![Plain Text in Terminal](images/rds-db-instance-creation-command-line-screenshot.png)
+
+With the method call to create and the method call to describe the DB instance, [her password was in plain text for the API request and response](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html#API_CreateDBInstance_Examples). That didn't seem very secure to Alice. 
+The [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) currently shows that the MasterUserPassword is a string:
```

Comment:
You need a section header like `#### ACK before Secrets Integration` above this sentence, since the custom resource you're describing here is what is *currently* produced for ACK using the `ack-generate apis rds` command :)